tests/yam/validate/validate_registered_addons.pm

```diff
@@ -15,7 +15,7 @@ use testapi;
 sub run {
     select_console 'root-console';
 
-    my @addon_list = split(/,/, get_var('ADDONS'));
+    my @addon_list = split(/,/, get_var('SCC_ADDONS'));
```

Comment:
Normally we use SCC_ADDONS for installation to register extensions and modules include recommended, while the test here only use ADDONS for extensions and modules to be tested, such as HA, PHUB, etc. I prefer to reserve ADDONS to avoid confusing.

By the way, I found existing test for HA missed ADDONS.